app/build.gradle

```diff
@@ -460,13 +464,10 @@ task downloadPlugins(
             into 'plugins/crate-azure-discovery'
         }
 
-        // copy crate-admin plugin
         copy {
-            from(project(':admin-ui').buildDir.path + '/crate-admin')
-            from(project(':admin-ui').buildDir.path + '/libs')
-            into 'plugins/crate-admin'
+            from("${buildDir}/crate-admin/")
+            into 'adminui'
```

Comment:
this is just about the resources for _site, right? so probably it would make sense to use the normal resource mechanism?